nixos/modules/services/networking/prosody.nix

```diff
@@ -672,6 +720,16 @@ in
         description = lib.mdDoc "Multi User Chat (MUC) configuration";
       };
 
+      pubsub = mkOption {
+        description = lib.mdDoc "Configures pubsub service";
+        type = types.nullOr (types.submodule pubsubOpts);
+        default = null;
+        example = {
+          domain = "pubsub.my-xmpp-example-host.org";
```

Comment:
```suggestion
          domain = "pubsubxmpp.example.org";
```

Reply:
In general it would... but the entire module should be ported over to that (which I have worked on and hopefully can open a PR for at some point)... so I'm fine with leaving this the way it currently is for now because of the rest of the module.